docs/source/conf.py

```diff
@@ -24,7 +24,7 @@ def setup(sphinx):
     sys.path.insert(0, os.path.abspath('utils'))  # travis
     sys.path.insert(0, os.path.abspath(opj(pardir, 'utils')))  # RTD
     from pygments_ansi_color import AnsiColorLexer
-    sphinx.add_lexer("ansi-color", AnsiColorLexer())
+    sphinx.add_lexer("ansi-color", AnsiColorLexer)
```

Comment:
Debian stable (buster) has 1.8.5 . I wonder if giving a class would work there, or we better pass instance if older sphinx here?

Reply:
tried with sphinx 1.8.5-8, it does fail with `TypeError: lex() argument must be a lexer instance, not a class`, so I will push a conditional here

Reply:
Thanks for checking.  I should have done more research.   However, the comment and code are inaccurate.  As mentioned in the commit message, v2.1 is when this change was made.  And despite being documented as to-be-removed-in-v3, it's still around.  I'll rework it.